iphone/Maps/UI/Settings/DrivingOptionsViewController.swift

```diff
@@ -14,6 +14,10 @@ class DrivingOptionsViewController: MWMTableViewController {
     ferryCrossingsCell.isOn = options.avoidFerry
     motorwaysCell.isOn = options.avoidMotorway
   }
+
+  override func tableView(_ tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
+    return nil
```

Comment:
Is it the correct fix? It looks like a workaround.

Reply:
It is kind of a hacky workaround for now if it works, as I couldn't find a proper way to do it.

Reply:
1. Please delete this method completely.
2. Please delete `headerTitle="AVOID ON EVERY ROUTE"` attribute from here: https://github.com/organicmaps/organicmaps/blob/master/iphone/Maps/UI/Settings/DrivingOptions.storyboard#L19